backend/routes/auth.js

```diff
@@ -227,7 +223,7 @@ const updateVolunteerUser = async (userId, request, response, next) => {
             }
             
             await volunteersQueries.updateVolunteer(formattedRequestBody);
-            if (request.file && request.user.f_picture && request.user.f_picture.includes('https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/')) {
+            if (request.file && user.v_picture && user.v_picture.includes('https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/')) {
                 storage.deleteFile(request.user.v_picture)
```

Comment:
what do this line and the same for the fellows on 116 do? if the request includes a new file to replace, delete the old one from the bucket?

Reply:
yes, it checks if the user is updating their picture, if yes and all goes well it deletes the old one from the cloud (only if it was already stored, because at first it starts with an empty string as default picture),
If an error or restriction prevents the update then the picture which was just stored gets deleted